### PR TITLE
sfwbar

### PR DESCRIPTION
**Pull request by elgithubo**
Adding sfwbar port

### CODE REVIEW

File: sfwbar/Makefile [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+CATEGORIES=	x11
+
+MAINTAINER=	tino.engel@mail.de
+COMMENT=	Flexible taskbar application for wayland compositors
+WWW=		https://github.com/LBCrion/sfwbar
+
+LICENSE=	GPLv3
+
+BUILD_DEPENDS=	wayland-protocols>0:graphics/wayland-protocols
+LIB_DEPENDS=	libgtk-3.so:x11-toolkits/gtk30 \
```
Comment (arrowd) [inline sfwbar/Makefile:12]: Some of these `LIB_DEPENDS` should be converted to `USES=gnome` and `USE_GNOME=...`

File: sfwbar/Makefile [hunk 2]
```diff
@@ -0,0 +1,32 @@
+PORTNAME=	sfwbar
+DISTVERSION=	v1.0_beta9
+CATEGORIES=	x11
+
+MAINTAINER=	tino.engel@mail.de
+COMMENT=	Flexible taskbar application for wayland compositors
+WWW=		https://github.com/LBCrion/sfwbar
+
+LICENSE=	GPLv3
```
Comment (arrowd) [inline sfwbar/Makefile:9]: Can you please add `LICENSE_FILE` knob too?

**Review by arrowd — CHANGES_REQUESTED** [review]

File: sfwbar/Makefile (hunk 1 above)
Comment (elgithubo) [reply sfwbar/Makefile]: Should be fixed.

File: sfwbar/Makefile (hunk 2 above)
Comment (elgithubo) [reply sfwbar/Makefile:9]: Has been added.

**Pull request merged**